src/error-handling/try-operator.md

```diff
@@ -19,27 +19,27 @@ some_expression?
 We can use this to simplify our error handling code:
 
 ```rust,editable
-use std::{fs, io};
-use std::io::Read;
+use std::fs;
+use std::io::{self, Error, ErrorKind};
```

Comment:
I would prefer not having to talk about this here — the slide is not about the standard library error types, it's about the `?` operator :smile:

Reply:
Yes, I have diverted from my original goal at the time I realise.

I wanted to illustrate that a function that returns `Option` can also leverage the `?` operator. But it is not a good example here. I will roll back the change and should I add a new slide instead to show the `Option` case ?

Reply:
I would start by putting it in the speaker notes like you do in this PR. You should try teaching the class, then you will see how much time there is for all the details (not a lot, in my experience :smile: )

Reply:
I am scheduled in 2 weeks :)

Reply:
Oh right, sorry that I missed that! I'll look forward to hear what you think of it!